lib/chippy_web/live/sprint_live/show.ex

```diff
@@ -11,6 +11,19 @@ defmodule ChippyWeb.SprintLive.Show do
     ChippyWeb.PageView.render("sprint_live.html", assigns)
   end
 
+  defp user_chip_counts(sprint_id) do
+    SprintServer.display_by_users(sprint_id)
+    |> Map.new(fn {name, projects} ->
+      {name, %{chip_count: projects |> Map.values() |> Enum.sum()}}
+    end)
+  end
+
+  defp online_users(sprint_id) do
+    Presence.list("users:" <> sprint_id)
+    |> Map.new(fn {name, %{metas: metas}} -> {name, %{device_count: length(metas)}} end)
+    |> Map.merge(user_chip_counts(sprint_id), fn _k, a, b -> Map.merge(a, b) end)
```

Comment:
I don't quite understand what this line is doing. I think it's taking the `user_chip_counts` map and then updating it with the map generated on the previous line, but I'm not seeing how `name` and `device_count` are mapping to `_k, a, b`.

Reply:
It's an oddity of how the 3rd argument to Map.merge works. Took me awhile to figure out as I originally just tried passing Map.merge as `fun`, though upon closer inspection I found it was in the docs. To resolve a conflict I guess the function gets the conflicting key and the two values to "resolve" (which here we just want to merge again).